.github/actions/setup-deploy/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: "Deploy prerequisites"
+description: "Prerequisites needed to deploy hubs in a 2i2c cluster"
+inputs:
+  provider:
+    description: "Cloud provider a cluster runs on"
+    required: true
+    default: "gcp"
+runs:
+  using: "composite"
+  steps:
+    - name: Checkout repo
+      uses: actions/checkout@v3
+
+    - name: Setup gcloud
+      uses: google-github-actions/setup-gcloud@v0
+      with:
+        version: "290.0.1"
+        # This is used for KMS only
+        project_id: two-eye-two-see
+        service_account_key: ${{ secrets.GCP_KMS_DECRYPTOR_KEY }}
+        export_default_credentials: true
+
+    - name: Get gcloud info
+      run: gcloud info
+      shell: bash
+
+    - name: Setup gcloud auth for docker
+      # FIXME: Add more auth providers & registries here as needed
+      run: |
+        gcloud auth configure-docker us-central1-docker.pkg.dev
+      shell: bash
```

Comment:
Perhaps @yuvipanda can answer this question?

Reply:
This is the commit, and it hasn't been changed since that: https://github.com/2i2c-org/infrastructure/commit/7aff26ea76777af0ef8aa9aef08c9fede824b9ba

I assume that we had previously built images and such before, but now we don't as part of deploy.

The image built seem to have been the custom 2i2c image, which is now available under quay.io publicly without need for authentication: https://github.com/2i2c-org/infrastructure/pull/925/files

I think we are safe to remove this section.